src/worker/work.rs

```diff
@@ -28,22 +28,22 @@ use super::Worker;
 
 static JOB_ID: AtomicUsize = AtomicUsize::new(0);
 
-pub fn spawn_worker(hash: H256, target: U256, worker: Box<Worker>, submit_port: u16) {
+pub fn spawn_worker(hash: H256, target: U256, worker: Box<Worker>, submit_port: u16, jobs: usize) {
```

Comment:
It seems `jobs` and `worker` both comes from `Config` object.
How about passing `config: Config` instead?

Reply:
I agree with you. I'll change it in the next patch.